wellcomecollection/wellcomecollection.org
# Exhibition Guides: Remove "audio with descriptions" from codebase

## What does this change?
[As discussed on Slack](https://wellcome.slack.com/archives/CUA669WHH/p1722260887043759), we never had "Audio with descriptions" guides and now the content type is deprecated, so I wanted to streamline what would become very busy files and completely remove traces of it.

## How to test

Run locally and try various types of EG to see if anything seems off. Try with Jason: http://localhost:3000/guides/exhibitions/Zdcs4BEAACMA6abC

## How can we measure success?

Nothing breaks and the code is tidier and easier to read through.

## Have we considered potential risks?
N/A
